pkgs/development/python-modules/certbot-dns-google/default.nix

```
@@ -24,9 +26,9 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
-  pytestFlagsArray = [ "-o cache_dir=$(mktemp -d)" ];
-
-  sourceRoot = "${src.name}/certbot-dns-google";
+  pytestFlagsArray = [
+    "-o cache_dir=$(mktemp -d)"
```

```suggestion
    "-o cache_dir=$(mktemp -d)"
    
    # Monitor https://github.com/certbot/certbot/issues/9606 for a solution
    "-W 'ignore:pkg_resources is deprecated as an API:DeprecationWarning'"
```
Don't we need this change?